src/views/navigation/globalComposerTab.js

```diff
@@ -3,13 +3,20 @@ import React from 'react';
 import Icon from 'src/components/icon';
 import Tooltip from 'src/components/tooltip';
 import { NavigationContext } from 'src/helpers/navigation-context';
+import { MIN_WIDTH_TO_EXPAND_NAVIGATION } from 'src/components/layout';
 import { AvatarGrid, AvatarLink, Label, IconWrapper } from './style';
 
 const GlobalComposerTab = () => {
+  const isWideViewport =
+    window && window.innerWidth > MIN_WIDTH_TO_EXPAND_NAVIGATION;
```

Comment:
This will break SSR rehydration any logged-in page I think? We should do this with CSS media queries inside the Tooltip component.